packages/toolchain/elemental-cli/collection.yaml

```diff
@@ -3,15 +3,15 @@ packages:
     name: "elemental-cli"
     bin_name: "elemental"
     category: "toolchain"
-    version: "0.20220427"
+    version: 0.20220428-1
     fips: false
     labels:
       github.repo: "elemental"
       github.owner: "rancher-sandbox"
       autobump.revdeps: "true"
       autobump.strategy: "git_hash"
       autobump.git.branch: "main"
-      git.hash: "ed32b2e85140abdb2217e46740804bc237aca4be"
```

Comment:
....sorry, I merged the other PR and I didn't noticed the bump was already there :facepalm:

Reply:
no problem, this should go smooth and pass all tests, since few Monday they are passing 😉 But it is good we waited to merge until elemental got few small details fixed.